### DISCUSSION TITLE
How get One record from collection by a unique column except id

### DISCUSSION THREAD

**Discussion by samimhamza-pgl** [pocketbase/pocketbase#2209] [Q&A] [answered]
For data migration I have stored id of mysql database as tmp_id in pocketbase collections, I have successfully migrated all other data except foreign keys. for updating foreign keys I need to get record by tmp_id which is unique

**Comment by ganigeorgiev** [comment, marked as answer]
You can use the List/Search web apis with `page` and `perPage` set to 1. Or use the `getFirstListItem` SDK helper:

```js
const myId = "something"
const record = await pb.collection("example").getFirstListItem(`tmp_id="${myId}"`);
```
